src/java/org/apache/cassandra/db/compaction/CompactionRealm.java

```diff
@@ -63,6 +63,15 @@ public interface CompactionRealm
      */
     Environment makeUCSEnvironment();
 
+    /**
+     * @return a {@link ShardManager} for this specific compaction realm. By default, this shard manager is ignored
```

Comment:
The comment does not sound correct. If this is not null, it will be used.

Maybe change "this shard manager is ignored" to "this is null"?

Reply:
Good catch, I updated the wording